packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.3
```

Comment:
Optional nit: Dart pre-release semver shifts everything to the right, so a bugfix would generally be 0.2.2+1; this is a feature version bump.

Reply:
Yea, I know.  I've just been treating each fix as a minor release.  The nuance didn't seem that important leading up to 1.0.